internal/codegen/codegen.go

```diff
@@ -0,0 +1,88 @@
+// Package codegen provides the code generation functionality for converting VSPEC signals to Go structs and ClickHouse tables.
+package codegen
+
+import (
+	_ "embed"
```

Comment:
I swear there's an exception for this one in the linter.

Reply:
I used to have the templates embedded in this file before I moved them to separate packages, so this import was an artifact.